decidim-elections/app/models/decidim/elections/election.rb

```diff
@@ -24,6 +24,11 @@ class Election < Elections::ApplicationRecord
 
       has_many :voters, class_name: "Decidim::Elections::Voter", inverse_of: :election, dependent: :destroy
       has_many :questions, class_name: "Decidim::Elections::Question", inverse_of: :election, dependent: :destroy
+      has_many :votes,
+               foreign_key: "decidim_election_id",
+               class_name: "Decidim::Elections::Vote",
+               through: :questions,
```

Comment:
Justr a question... is not working using : `counter_cache: true`, instead of having `update_votes_count` method ?

Reply:
Ok, let's leave it like this for the moment.